libdnf/dnf-context.c

```diff
@@ -190,11 +227,55 @@ dnf_context_init(DnfContext *context)
                                                   g_free, g_free);
     priv->user_agent = g_strdup("libdnf/" PACKAGE_VERSION);
 
+    g_autoptr(PeasEngine) engine = peas_engine_get_default ();
+
+    /* Enable Python 3 plugins */
+    peas_engine_enable_loader (engine, "python3");
```

Comment:
But this is exactly what I don't want for rpm-ostree.  Why can't python plugins stay as dnf plugins?